docs/source/concepts/architecture/xarray-standards.rst

```diff
@@ -37,16 +37,41 @@ Xarray Standard Attributes
 The following standard attributes are used internally to GeoIPS for consistent
 generation of titles, legends, regridding, etc
 
-* 'source_name' - REQUIRED
-* 'platform_name' - REQUIRED
-* 'data_provider' - REQUIRED
-* 'start_datetime' - REQUIRED
-* 'end_datetime' - REQUIRED
+Note these are referenced explicitly in the geoips/geoips_utils.py
```

Comment:
@jsolbrig what are your thoughts on this verbiage?

Reply:
Yeah, this is kind of weird. We should think of a better way to describe this that doesn't require referencing a specific file in the code.

We should develop a new document for this as we develop our DataTree standards anyway, though...